src/app/beer_garden/api/http/handlers/vbeta/file.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+import io
+
+from brewtils.resolvers.parameter import BYTES_PREFIX
+
+from beer_garden.api.http.authorization import Permissions, authenticated
+from beer_garden.api.http.base_handler import BaseHandler
+from beer_garden.db.mongo.models import RawFile
+
+
+class RawFileAPI(BaseHandler):
+    @authenticated(permissions=[Permissions.READ])
+    async def get(self):
+        """
+        ---
+        summary: Retrieve a File
+        parameters:
+          - name: file_id
+            in: query
+            required: true
+            description: The file ID
+            type: string
+        responses:
+          200:
+            description: The requested File or FileChunk data
+            schema:
+              $ref: '#/definitions/FileStatus'
+          404:
+            $ref: '#/definitions/404Error'
+          50x:
+            $ref: '#/definitions/50xError'
+        tags:
+          - Files
+        """
+        db_file = RawFile.objects.get(id=self.get_argument("file_id", default=""))
+        file = db_file.file.read()
+
+        self.set_header("Content-Type", "application/octet-stream")
+        self.write(file)
```

Comment:
Again, this could be a problem for large files, it'd be nice to have another API for "streaming" chunks back. I think GridFS already stores thinks in chunks (at every 4MB) so maybe that's our chunk size?